src/trips/trip.repository.ts

```diff
@@ -29,7 +29,7 @@ export class TripRepository extends Repository<Trip>{
         throw new UnauthorizedException('Your account must be approved.');
       }
       query = this.createQueryBuilder('trip');
-      query.where('trip.userId = :userId', { userId: user.id });
+      query.where('trip.userId = :userId', { user: user.id });
```

Comment:
wouldn't this break the query?
you're searching for trips that have userId equal to userId but you're now passing user instead of userId

i think in this case will not find any always

Reply:
@abdallahdak7 ^^^

Reply:
fixed